graphics/brlcad/brlcad.info

```diff
@@ -1,8 +1,8 @@
 PRGNAM="brlcad"
-VERSION="7.34.2"
+VERSION="7.36.0"
 HOMEPAGE="https://brlcad.org"
-DOWNLOAD="https://github.com/BRL-CAD/brlcad/archive/rel-7-34-2/brlcad-7.34.2.tar.gz"
-MD5SUM="62615c0004836c4125b052e64569d74b"
+DOWNLOAD="https://github.com/BRL-CAD/brlcad/archive/rel-7-36-0/brlcad-7.36.0.tar.gz"
```

Comment:
I think this should be (see https://slackbuilds.org/GITHUB_URLs.txt):

https://github.com/BRL-CAD/brlcad/archive/rel-7-36-0/brlcad-rel-7-36-0.tar.gz

if you want to get the same tarball with wget and with something that respects the content-disposition header, like downloading with a browser.

The script does have the double file check but it does not work - it tries to untar one path with `tar.bz2` and the other with `tar.gz`.

If you change DOWNLOAD to the link above it works with or without content-disposition being respected if you change the build script to this:

```diff
diff --git a/graphics/brlcad/brlcad.SlackBuild b/graphics/brlcad/brlcad.SlackBuild
index 4cff4e8d96..8ad525bfc9 100644
--- a/graphics/brlcad/brlcad.SlackBuild
+++ b/graphics/brlcad/brlcad.SlackBuild
@@ -57,9 +57,9 @@ set -e
 rm -rf $PKG
 mkdir -p $TMP $PKG $OUTPUT
 cd $TMP
-rm -rf $PRGNAM-$VERSION $PRGNAM-$SRCVER
-tar xvf $CWD/$PRGNAM-$VERSION.tar.bz2 || tar xvf $CWD/$PRGNAM-$SRCVER.tar.gz
-cd $PRGNAM-$VERSION || cd $PRGNAM-$SRCVER
+rm -rf $PRGNAM-$SRCVER
+tar xvf $CWD/$PRGNAM-$SRCVER.tar.gz
+cd $PRGNAM-$SRCVER
 chown -R root:root .
 find -L . \
  \( -perm 777 -o -perm 775 -o -perm 750 -o -perm 711 -o -perm 555 \
```